.buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh

```diff
@@ -5,10 +5,10 @@ set -euo pipefail
 source .buildkite/env-scripts/util.sh
 
 echo "--- Installing kind & kubectl"
-retry_with_count 5 .buildkite/deploy/kubernetes/scripts/install-kind.sh
-retry_with_count 5 .buildkite/deploy/kubernetes/scripts/install-kubectl.sh
+retry -t 5 -- .buildkite/deploy/kubernetes/scripts/install-kind.sh
+retry -t 5 -- .buildkite/deploy/kubernetes/scripts/install-kubectl.sh
 
-echo "--- Setting up kind"
+echo "Setting up kind"
```

Comment:
I'd suggest that we keep `---` (or maybe it could be changed to `~~~` to keep it folded); plz refer to https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output